docs/content/docs/how-to/creating-tables.md

```diff
@@ -114,6 +114,78 @@ Partition keys must be a subset of primary keys if primary keys are defined.
 By configuring [partition.expiration-time]({{< ref "docs/maintenance/manage-partition" >}}), expired partitions can be automatically deleted.
 {{< /hint >}}
 
+## Create Table As
+
+Tables can also be created and populated by the results of a query. 
+
+{{< tabs "create-table-as" >}}
+
+{{< tab "Flink" >}}
+
+// Flink will create target table first and then start a `INSERT INTO target_table SELECT * FROM source_table` job,
+// for batch mode, the job will exit when the job finished, for streaming mode, the job will not exit.
```

Comment:
It doesn't need to explain the job behavior for batch mode and streaming mode.

Reply:
For spark,it has not problems ,after the operation, we can get the correct result ,and read the correct the data from the target table, but for flink. the default mode is streaming mode ,  we need to enable the checkpoint , but if the user forgot to enable the checkpoint , we will can not get data from the target table , user will be confused , why the same operation, it is fine for spark ,but it is not fine for flink ,so I add the explanation

Reply:
@zhangjun0x01, because the explanation is in the `Flink` tab, you could explain that enable the checkpoint for streaming mode, no need to explain the job behavior.